vevestaX/vevesta.py

```diff
@@ -431,10 +431,10 @@ def __EDA(self, fileName):
                 nonNumericFeatureImage.anchor = columnTextImgone
                 nonNumericPlotSheet.add_image(nonNumericFeatureImage)
 
-            if os.path.exists(os.path.join(directoryToDumpData,FeatureDistributionImageFile)):
-                workBook.create_sheet('Feature Distribution')
-                featureDistribution = workBook['Feature Distribution']
-                featureDistributionImage = openpyxl.drawing.image.Image(os.path.join(directoryToDumpData,FeatureDistributionImageFile))
+            if os.path.exists(os.path.join(directoryToDumpData,FeatureHistogramImageFile)):
+                workBook.create_sheet('EDA-Feature Histogram')
```

Comment:
Since EDA-Feature histogram is used twice..the right way would be to make a variable workBookName and assign it this value and then use the variable in both these places